modules/ecs/cluster/variables.tf

```diff
@@ -32,4 +32,28 @@ variable "tags" {
   default = {
     terraform = true
   }
+}
+
+variable "app_name" {
+  description = "Name of the app"
+  type        = string
+  default     = ""
+}
+
+variable "current_account_id" {
+  description = "The AWS account number where the ecs cluster is"
+  type        = string
+  default     = ""
+}
+
+variable "role_name" {
+  description = "The name of the role for ECS"
+  type        = string
+  default     = "ECSRuntimeRole"
+}
+
+variable "kms_key_arn" {
+  description = "The arn of the kms key"
+  type        = string
+  default     = ""
```

Comment:
This default will break the instances of this module in `app/ecs.tf` and `billing/ecs.tf` we should either update this to be required or also set these values when this module is used.